Makefile

```diff
@@ -99,10 +101,22 @@ livedoc:
 	$(MAKE) doc
 	python3 -m http.server 8888 --directory doc/_build
 
-package := atd
+# Prepare the opam files for a release. They're derived from 'dune-project'
+# by 'dune build' and then patched to work around some issues.
+# https://discuss.ocaml.org/t/when-exactly-does-dune-generate-opam-files/8292/2
+#
+.PHONY: opam-files
+opam-files:
+	$(DUNE) build
+	./scripts/patch-opam-files
+
+# This is only part of the release process.
+# See complete release instructions in CONTRIBUTING.md.
+#
 .PHONY: opam-release
 opam-release:
-	dune-release distrib --skip-build --skip-lint --skip-tests -n $(package)
-	dune-release publish distrib --verbose -n $(package)
-	dune-release opam pkg -n $(package)
-	dune-release opam submit -n $(package)
+	dune-release tag
+	dune-release distrib
+	dune-release publish
+	dune-release opam pkg
+	dune-release opam submit
```

Comment:
this is the same as running `dune-release bistro` but maybe it's better to be explicit

Reply:
ah, I didn't know this. The help page assumes we'd figure this out.
```
       bistro
           For when you are in a hurry or need to go for a drink
```
well, I guess we can leave the makefile as it it. I'll probably change my mind if I have to run this a lot.